src/main/drivers/accgyro_spi_mpu6500.c

```diff
@@ -99,10 +99,14 @@ static bool mpu6500Detect(void)
     uint8_t tmp;
 
     mpu6500ReadRegister(MPU6500_RA_WHOAMI, &tmp, 1);
+#ifdef COLIBRI_RACE
```

Comment:
yes, this needs to be fixed.  Is the MPU hardware you're using returning something different?